scripts/adopters/index.ts

```diff
@@ -0,0 +1,11 @@
+import { dirname, resolve } from 'path';
+import { fileURLToPath } from 'url';
+
+import { writeJSON } from '../utils/readAndWriteJson';
+
+const currentFilePath = fileURLToPath(import.meta.url);
```

Comment:
What is `import.meta.url` here?

Reply:
It is an alternative way of getting the current `currentFilePath `and `currentDirPath`. Apprently `__dirname` and and `__filename` are not in available in es modules. 

### Value of `import.meta.url` in `/scripts/index.ts` 
![image](https://github.com/user-attachments/assets/63f365a2-2d8e-4ca5-bf6a-9b04eb933c09)

Reference: [Alternative for __dirname in Node.js when using ES6 modules](https://stackoverflow.com/a/50052194/13098662)